lib/udp.js

```diff
@@ -88,7 +88,7 @@ function SyslogStream(opts) {
     this.facility = opts.facility || 16;
     this.name = opts.name || process.title || process.argv[0];
     this.writable = true;
-
+    this.messagePrefix = (opts.ceeCookie || false ? '@cee: ' : '');
```

Comment:
`ceeCookie` is very WMF-specific. I'd prefer to call it simply `prefix` allowing the external user code to set the prefix (and we could then set it to `@cee: `, so that we end up with something like:

```js
this.messagePrefix = opts.prefix || '';
```

Reply:
Is @CEE not dervied from [this project](https://cee.mitre.org/about/ )? Why do think this is WMF specific? Are there other structured logging standards? I would prefer the simple binary choice over a free-text prefix. @Pchelolo what do you think?

Reply:
Sorry, I meant to say that the prefix should be general in nature, it just so happens that we are using `@cee` in production, but other entities might want to use something different, and it's better to give everybody the option rather than stating *you can either use this specific prefix or no prefix at all*.